package.json

```diff
@@ -16,60 +16,61 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build",
+    "build": "ember build --environment=production",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember server",
     "test": "ember test",
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "ember-auto-import": "^1.5.3",
-    "ember-cli-babel": "^7.13.0",
-    "ember-cli-htmlbars": "^4.2.0",
+    "ember-auto-import": "^1.6.0",
+    "ember-cli-babel": "^7.17.2",
+    "ember-cli-htmlbars": "^5.2.0",
     "git-repo-version": "^1.0.2",
     "lodash.merge": "^4.6.1",
     "rollbar": "~2.14.4"
   },
   "devDependencies": {
-    "@ember/optional-features": "^1.1.0",
+    "@ember/optional-features": "^1.3.0",
     "@fortawesome/ember-fontawesome": "^0.2.1",
-    "@fortawesome/free-brands-svg-icons": "^5.12.0",
-    "@glimmer/component": "^1.0.0",
-    "babel-eslint": "^10.0.3",
+    "@fortawesome/free-brands-svg-icons": "^5.14.0",
+    "@glimmer/component": "^1.0.1",
+    "@glimmer/tracking": "^1.0.0",
```

Comment:
Why tracking landed here? :)

Reply:
It's added automatically when running `ember-cli-update --to 3.16.2`

I can remove it?